src/ROUTES.ts

```diff
@@ -53,6 +53,11 @@ const ROUTES = {
         getRoute: (query: string, reportID: string) => `search/${query}/view/${reportID}` as const,
     },
 
+    TRANSACTION_HOLD_REASON_RHP: {
+        route: '/search/:query/hold/:transactionID/hash/:searchHash',
+        getRoute: (query: string, transactionID: string, searchHash: number) => `search/${query}/hold/${transactionID}/hash/${searchHash}` as const,
```

Comment:
NAB we can remove the `hash` string from the route
```suggestion
        getRoute: (query: string, transactionID: string, searchHash: number) => `search/${query}/hold/${transactionID}/${searchHash}` as const,
```

Reply:
I did this change as you asked, but I personally prefer more descriptive urls, so I would keep it.
Works the same either way